src/python/bcc/__init__.py

```diff
@@ -62,13 +62,47 @@ def resolve_name(self, name):
             return -1
         return addr.value
 
+class BPFPerfType:
+    # From perf_type_id in uapi/linux/perf_event.h
+    HARDWARE = 0
+    SOFTWARE = 1
```

Comment:
Are `PERF_TYPE_HW_CACHE` and `PERF_TYPE_RAW` not supported by the underlying implementation? It could be nice to have these as well.